packages/fiori/src/themes/IllustratedMessage.css

```diff
@@ -47,7 +47,8 @@
     top: 0;
 }
 
-.ui5-illustrated-message-title {
+.ui5-illustrated-message-title :first-child,
```

Comment:
use [ui5-title] instead of first child selector. we always know what the content is.